vncdotool/api.py

```diff
@@ -37,7 +37,7 @@ class TimeoutError(OSError):
         pass
 
 
-def connect(server, password=None):
+def connect(server, password=None, factory_class=None, proxy=None, timeout=None):
```

Comment:
could we do use default values for the kwargs rather than conditionally set inline in the function body?

```
def connect(server, password=None, factory_class=VNCDoToolFactory, proxy=ThreadedVNCClientProxy, timeout=None):
```

Reply:
Updated. It required I reorder the code so that the proxy class is defined.